dev/pnpm.ts

```diff
@@ -0,0 +1,872 @@
+// GENERATORS
+const searchGenerator: Fig.Generator = {
+  script: function (context) {
+    if (context[context.length - 1] === "") return "";
+    const searchTerm = context[context.length - 1];
+    return `curl -s -H "Accept: application/json" "https://api.npms.io/v2/search?q=${searchTerm}&size=20"`;
+  },
+  postProcess: function (out) {
+    try {
+      var temp = JSON.parse(out).results.map(
+        (item) =>
+          ({
+            name: item.package.name,
+            description: item.package.description,
+          } as Fig.Suggestion)
+      ) as Fig.Suggestion[];
+    } catch (e) {
+      return [];
+    }
+
+    return temp;
```

Comment:
Let's remove the `var` and return directly the output of `map`:
```suggestion
      return JSON.parse(out).results.map(
        (item) =>
          ({
            name: item.package.name,
            description: item.package.description,
          } as Fig.Suggestion)
      ) as Fig.Suggestion[];
    } catch (e) {
      return [];
    }
```